### PR TITLE
💂 Add better description for authentication.

### PR DESCRIPTION
**Pull request by mxbaylee**
* Breaks up each auth by type
* Removes keys that aren't useful for all type

image
image

### CODE REVIEW

File: api-playground/mdx/authentication.mdx [hunk 1]
```diff
@@ -28,36 +30,58 @@ authMethod: "bearer"
...
     "auth": {
-        "method": "basic",
-        "name": "user-id:user-key"
+        "method": "basic"
     }
 }
 ```
 
-You can also request a single API key by excluding the colon.
+```md page metadata
```
Comment (hahnbeelee) [inline api-playground/mdx/authentication.mdx]: could we capitalize all instances of `Page Metadata`?

Comment (mxbaylee) [reply api-playground/mdx/authentication.mdx]: yes! globally or just this diff?

image

Comment (hahnbeelee) [reply api-playground/mdx/authentication.mdx]: just the ones that are "```md page metadata" on this page

Comment (mxbaylee) [reply api-playground/mdx/authentication.mdx]: updated!

image

**Review by rpmccarter — APPROVED** [review]
nice

**Pull request merged**